lib/tasks/data_export.rake

```diff
@@ -8,10 +8,10 @@ namespace :core do
 
   desc "Export all data XMLs for import into Central Data System (CDS)"
   task :full_data_export_xml, %i[year] => :environment do |_task, args|
-    collection_year = args[:year].present? ? args[:year].to_i : nil
+    collection = args[:year].present? ? args[:year].to_i : nil
```

Comment:
If this might be "users" now, presumably we should not cast it to a number? (Also maybe rename year?)

Reply:
Good shout! should also be updated here [Move method and update task argument](https://github.com/communitiesuk/submit-social-housing-lettings-and-sales-data/pull/2543/commits/969e65ad82760decaecfacaa4a2f8769344db168)